app/assets/stylesheets/barnardos/_textarea.scss

```diff
@@ -62,6 +62,9 @@
 .textarea__hint {
   color: $secondary-text;
   display: block;
+  font-family: $font-family;
```

Comment:
you'd have to make some fake markup in gallery or something then recreate setting the label to have a title like style